rasa/core/constants.py

```diff
@@ -57,4 +57,5 @@
 UTTER_PREFIX = "utter_"
 RESPOND_PREFIX = "respond_"
 
+DIALOGUE = "dialogue"
```

Comment:
Can we make this constant name a bit more expressive? From seeing this I have no idea where this could be used and for what.